src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -12,26 +12,32 @@
 import org.jdbi.v3.sqlobject.SqlObjectPlugin;
 import org.jdbi.v3.testing.JdbiRule;
 import org.junit.Before;
-import org.junit.Rule;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class DatasetDaoTest {
   static final Logger logger = LoggerFactory.getLogger(DatasetDaoTest.class);
-  private DatasetDao datasetDAO;
-  private marquez.service.models.Namespace namespace = Generator.genNamespace();
-  private NamespaceDao namespaceDAO;
+  private static DatasetDao datasetDAO;
+  private static marquez.service.models.Namespace namespace;
+  private static NamespaceDao namespaceDAO;
 
-  @Rule
-  public final JdbiRule dbRule =
+  @ClassRule
+  public static final JdbiRule dbRule =
```

Comment:
`static` variables and methods aren't associated with an instance of `DatasetDaoTest`, is that what we want? That is, a single copy of `JdbiRule` we be shared across `DatasetDaoTest`, which for tests isn't techincally a problem, just redunt, so I would define `JdbiRule ` and `setup()` as non-static and revert the change.

Reply:
It's required to be `static` b/c the test class setup method, where it's used, is static.

Reply:
OOh, right. Sorry, disregard my comment.